### PR TITLE
Automation Manager Web App

### PR DESCRIPTION
**Pull request by deshanigtk**
## Automation tests
 > N/A

## Security checks
 > N/A

### CODE REVIEW

File: internal/automation-manager/web-app/src/main/resources/global.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+automation.manager.host=localhost
+automation.manager.port=9766
+automation.manager.https-port=8243
+static-scanner.start-scan=am/1.0.0/staticScanner/startScan
+dynamic-scanner.start-scan=am/1.0.0/dynamicScanner/startScan
+get.static.scanners=am/1.0.0/getStaticScanners
+get.dynamic.scanners=am/1.0.0/getDynamicScanners
+webapp.client-id=04K_et0E9wyIKgMFOqFfGSDO6ska
+webapp.client-secret=OhH094zA7CizfpCfOTBO63CMwa4a
```
Comment (dulanjal) [inline internal/automation-manager/web-app/src/main/resources/global.properties]: Use placeholders for clientid and secret

File: internal/automation-manager/web-app/src/main/resources/org.wso2.security.tools.am.webapp/api_manager.crt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```
Comment (dulanjal) [inline internal/automation-manager/web-app/src/main/resources/org.wso2.security.tools.am.webapp/api_manager.crt]: Why .crt files are needed? Ideally those should be read from the truststore.

**Comment by maheshika** [comment]
staled

File: internal/automation-manager/web-app/src/main/resources/global.properties (hunk 1 above)
Comment (deshanigtk) [reply internal/automation-manager/web-app/src/main/resources/global.properties]: Replaced hard coded values with placeholders

**Pull request merged**